QYWSR3/TodoListApplication/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using ToDoListApplication.Commands;
+
+internal class Program
+{
+    static int Main(string[] args)
+    {   
+        Console.WriteLine("|------------------------------------------------|");
+        Console.WriteLine("|       TODO-list application WELCOMES YOU!      |");
+        Console.WriteLine("|------------------------------------------------|");
+        try
+        {
+            new CommandRunner().Run();
+            return 0;
+        }
+        catch (Exception ex)
```

Comment:
Kivételkezelés itt is okés, de lehet felhasználóbarátabb lett volna a command futtató belsejébe.